src/main/java/ch/njol/skript/expressions/ExprExperience.java

```diff
@@ -95,42 +98,45 @@ public Class<?>[] acceptChange(final ChangeMode mode) {
 	
 	@Override
 	public void change(final Event e, final @Nullable Object[] delta, final ChangeMode mode) {
```

Comment:
Refractor `e` to `event`

Reply:
I think this is outside the scope of this PR. See [comment](https://github.com/SkriptLang/Skript/pull/5389#discussion_r1084344129)